test/user_defined_output/fake_plugin/fake_user_defined_output_test.cc

```diff
@@ -0,0 +1,51 @@
+#include "envoy/registry/registry.h"
+
+#include "external/envoy/source/common/config/utility.h"
+
+#include "test/test_common/proto_matchers.h"
+
+#include "test/user_defined_output/fake_plugin/fake_user_defined_output.h"
+#include "test/user_defined_output/fake_plugin/fake_user_defined_output.pb.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+namespace {
+
+using ::nighthawk::FakeUserDefinedOutput;
+using ::nighthawk::FakeUserDefinedOutputConfig;
+using ::testing::HasSubstr;
+
+TEST(FakeUserDefinedOutputPluginFactory, CreateEmptyConfigProtoCreatesCorrectType) {
```

Comment:
More tests should probably be created for this fake plugin, but I wanted to wait until we actually agree on the API first before making those tests.

Reply:
The API looks good. Can we expand the tests coverage for the plugin?

Reply:
Done.